cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh

```diff
@@ -18,7 +18,7 @@ fi
 SRIOV_TESTS_NS="${SRIOV_TESTS_NS:-kubevirt-test-default1}"
 
 function set_kind_params() {
-    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-quay.io/kubevirtci/kindest_node:v1.17.0}"
+    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.17.0}"
```

Comment:
Leftovers from a rebase that went not correct, please check it

Reply:
Removed.